defaults/main.yml

```diff
@@ -23,6 +23,7 @@ besu_data_dir: "{{ besu_base_dir }}/data"
 besu_log_dir: "/var/log/besu"
 besu_log4j_config_file: ""
 besu_profile_file: "/etc/profile.d/besu-path.sh"
+besu_engine_jwt_secret: "{{ besu_config_dir }}/jwt-secret.hex"
```

Comment:
It is just ignored, but it is cleaner to not create it by default, since not all configurations need it, so I have reworked the PR to leave it unset by default, please recheck it and thanks for reporting!